### ISSUE TITLE
Hard to read in the metro

### ISSUE THREAD

**Issue by cesine** [jrwdunham/dative#286]
![screenshot_2016-01-12-09-37-13](https://cloud.githubusercontent.com/assets/196199/12266743/da17bed8-b912-11e5-8f75-4e7b3365db9e.png)

**Comment by jrwdunham** [comment]
Yes, I don't know what to do about mobile yet. Dative only really works on desktop browsers right now. Suggestions/contributions on making it mobile-friendly would be very much appreciated...
